src/main/resources/institutional-config.properties

```diff
@@ -141,6 +142,7 @@ property.files=classpath:org/kuali/kfs/krad/ApplicationResources.properties,\
   classpath:edu/cornell/kfs/rass/cu-rass-resources.properties,\
   classpath:edu/cornell/kfs/tax/cu-tax-resources.properties,\
   classpath:edu/cornell/kfs/concur/cu-concur-resources.properties,\
+  classpath:edu/cornell/kfs/pmw/cu-pmw-resources.properties,\
```

Comment:
There are a number of parameters in CU-ApplicationResources.properties, we should create a follow up user story to move those constants to this.

Reply:
I've created KFSPTS-32594 to get the properties moved over accordingly.